webdriver/tests/bidi/integration/cookies_with_network_events.py

```diff
@@ -111,10 +111,17 @@ async def test_fetch(
     wait_for_future_safe,
     url,
     domain_1,
+    inline
 ):
     # Clean up cookies in case some other tests failed before cleaning up.
     await bidi_session.storage.delete_cookies()
 
+    # Navigate away from about:blank to make sure document.cookies can be used.
+    await bidi_session.browsing_context.navigate(
+        context=new_tab["context"], url=inline("<div>foo</div>"),
+        wait="complete"
+    )
+
     cookie_name = "foo"
     cookie_value = "bar"
     # Add `Access-Control-Allow-Origin` header for cross-origin request to work.
```

Comment:
is it still remaining a cross-origin request?

Reply:
It should have been, but changed to use `url` fixture instead, that has the argument `domain` with default empty string.